package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
+    "build": "skyux build-public-library",
+    "start": "skyux serve"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/animations": ">=7.2.2",
+    "@angular/common": ">=7.2.2",
+    "@angular/compiler": ">=7.2.2",
+    "@angular/core": ">=7.2.2",
+    "@angular/forms": ">=7.2.2",
+    "@angular/http": ">=7.2.2",
+    "@angular/platform-browser": ">=7.2.2",
+    "@angular/platform-browser-dynamic": ">=7.2.2",
+    "@angular/router": ">=7.2.2",
+    "@blackbaud/auth-client": ">=2.13.0",
+    "@skyux/assets": ">=3.0.0",
+    "@skyux/config": ">=3.5.0",
+    "@skyux/core": ">=3.5.2",
+    "@skyux/http": ">=3.2.0",
+    "@skyux/i18n": ">=3.5.2",
+    "@skyux/omnibar-interop": ">=3.1.0",
+    "@skyux/router": ">=3.1.1",
+    "@skyux/theme": ">=3.4.1",
+    "rxjs": ">=6.3.3",
+    "rxjs-compat": ">=6.3.3",
+    "zone.js": ">=0.8.28"
+  },
+  "dependencies": {
+    "jwt-decode": "^2.2.0"
```

Comment:
It's recommended to set `dependencies` to specific versions (without the caret `^`).